etc/nginx/ssl/README.md

```diff
@@ -0,0 +1,7 @@
+# SSL support for the local build
+
+This folder will contain self-signed certificates for SSL support in local development.
+
+They are generated/updated each time the local Docker stack is upped using `etc/nginx/ssl/gen-cert.sh`
```

Comment:
@limed Can you update this line of the README please? It looks like it refers to the old/broken way I initially tried

Reply:
Actually this is the update method, the `gen-cert.sh` script is similar to the old way but the content of the script is different